src/libraries/Common/src/System/Security/Cryptography/SLHDsa.cs

```diff
@@ -188,17 +100,16 @@ public int SignData(ReadOnlySpan<byte> data, Span<byte> destination, ReadOnlySpa
                     SR.Argument_SignatureContextTooLong255);
             }
 
-            if (destination.Length < SignatureSizeInBytes)
+            int signatureSizeInBytes = Algorithm.SignatureSizeInBytes;
+            if (destination.Length < signatureSizeInBytes)
```

Comment:
The prevailing style in .NET crypto code (or, if you prefer, "Jeremy's style") is that there's a blank line before any control-flow statement that isn't preceded by a `{`.

```suggestion
            int signatureSizeInBytes = Algorithm.SignatureSizeInBytes;

            if (destination.Length < signatureSizeInBytes)
```